### ISSUE TITLE
Problem with Always on Top option

### ISSUE THREAD

**Issue by vsub** [winsiderss/systeminformer#1513]
### Brief description of your issue

If the option is enabled,almost all windows that come from SytemInformer appear behind the main window so you have to either minimize the main window or move it

Basically the affected window don't appear with AlwaysOnTop status

### Expected behavior (optional)

All windows should appear infront of the main window

**Comment by dmex** [comment]
Duplicate of #200

----------

The Always on Top bug fixes were blocked on #1572 and should be resolved later this week 👍
